lib/known_bugs.pm

```diff
@@ -54,4 +55,15 @@ sub create_list_of_serial_failures {
     return $serial_failures;
 }
 
+
+# check_journal module=>'sshd'
+sub check_journal {
+    my (%args) = @_;
+    my $module = $args{module};
+    my $command = ($module ? "journalctl -u $module" : "journalctl" );
+
+    select_console 'root-console';
+    assert_script_run "$command >> /dev/$serialdev \n";
```

Comment:
Won't a call to `assert_script_run` be fatal in case it fails?. Even if the intention is running it on a post fail hook, killing the test because of this seems a bit too much

Reply:
It will fail, but I am not sure if this kills the test completely? @okurz do you know?